.github/workflows/linux-eic-shell.yml

```diff
@@ -884,7 +884,7 @@ jobs:
 
   trigger-container:
     runs-on: ubuntu-latest
-    if: ${{ github.event_name != 'merge_group' && github.actor != 'dependabot[bot]' }}
+    if: ${{ github.actor != 'dependabot[bot]' }}
```

Comment:
While we're here:
```suggestion
    env: 
      secret: ${{ secrets.EICWEB_CONTAINER_TRIGGER }}
    if: ${{ github.actor != 'dependabot[bot]' && secret != '' }}
```

Reply:
Why can't we compare the value directly?

Reply:
Okay, I was confused, this is not a `step.*.if`, but a `jobs.*.if`, where `env` should not be accessible, allegedly.